sklearn/cluster/hierarchical.py

```diff
@@ -467,7 +467,7 @@ def linkage_tree(X, connectivity=None, n_clusters=None, linkage='complete',
             return children_, 1, n_samples, None, distances
         return children_, 1, n_samples, None
 
-    connectivity, n_components = _fix_connectivity(X, connectivity,
+    connectivity, n_connected_components = _fix_connectivity(X, connectivity,
                                                    affinity=affinity)
```

Comment:
This is now not correctly indented

Reply:
I'm puzzled, why is this not caught by circleci/lint?

Reply:
Fixed in commit [#3e0536a](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/3e0536ae2157575e0af53a9124f6407f6cfd1439)